### PR TITLE
ui components: InfoTooltip

### PR DESCRIPTION
**Pull request by chasulin**

### CODE REVIEW

File: packages/ui-components/src/styles/theme/components.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/** MUI theme components */
+
+import { createTheme } from "@mui/material/styles";
+
+const referenceTheme = createTheme();
+
+const components = {
+  MuiTooltip: {
```
Comment (chasulin) [inline packages/ui-components/src/styles/theme/components.ts:9]: this just gives us a good default + can be overwritten/customized in any project repo's `components.ts` file

File: packages/ui-components/src/components/InfoTooltip/InfoTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  const handleClose = () => {
+    if (open) {
+      setOpen(false);
+    }
+  };
+
+  return (
+    <Tooltip
+      {...otherProps}
```
Comment (pnavarrc) [inline packages/ui-components/src/components/InfoTooltip/InfoTooltip.tsx]: Consider moving this to the end of the component to allow overriding the defaults

```tsx
<Tooltip
  open={open}
  {/* ... */}
  {...otherProps}
>
// ...
</Tooltip>
```

**Review by pnavarrc — COMMENTED** [review]
Added a small note!

**Review by faihegberg — APPROVED** [review]

**Comment by chasulin** [comment]
close #53

**Pull request merged**